docs/stacks-guide-azure.md

```diff
@@ -0,0 +1,429 @@
+    family: Gen5
+  storageProfile:
+    storageGB: 25
+    backupRetentionDays: 7
+    geoRedundantBackup: false
+  providerRef:
+    name: demo-azure
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+---
+apiVersion: compute.azure.crossplane.io/v1alpha2
+kind: AKSClusterClass
+metadata:
+  name: azure-aks-standard
+  namespace: azure-infra-dev
+specTemplate:
+  resourceGroupName: demo-rg
+  vnetSubnetID: /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/demo-rg/providers/Microsoft.Network/virtualNetworks/demo-vnet/subnets/demo-subnet
+  location: Central US
+  version: "1.12.8"
+  nodeCount: 1
+  nodeVMSize: Standard_B2s
+  dnsNamePrefix: crossplane-aks
+  disableRBAC: false
+  writeServicePrincipalTo:
+    name: akscluster-net
+  providerRef:
+    name: demo-azure
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+EOF
+
+kubectl apply -f environment.yaml
+```
+
+The steps that we have taken so far have been related to things that can be
+shared by all resources in all namespaces of that Crossplane cluster. Now, we
+will keep going with an app namespace which we will populate with resources that
+will help Crossplane know with what configuration it should satisfy the claims.
+You can use any namespace for your app's resources but for this tutorial we'll
+use the `app-project1-dev` namespace we created.
+
+Now we need to tell Crossplane which resource classes should be used to satisfy
+our claims in that app namespace. We will create portable classes that have have
+reference to non-portable ones that we created earlier. In our claims, we can
+refer to those portable classes directly or label one as the default portable
+class to be used in claims that do not have class reference.
+
+For example, MySQLInstanceClass is a portable class that can refer to Azure's
+SQLServerClass, which is a cloud-specific class.
+
+```
+cat > namespace.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: standard-mysql
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+---
+apiVersion: compute.crossplane.io/v1alpha1
+kind: KubernetesClusterClass
+metadata:
+  name: standard-cluster
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: AKSClusterClass
+  apiVersion: compute.azure.crossplane.io/v1alpha2
+  name: azure-aks-standard
+  namespace: azure-infra-dev
+---
+EOF
+
+kubectl apply -f namespace.yaml
+```
+
+For more details about what is happening behind the scenes, read more about
+[portable claims in Crossplane][portable-claims].
+
+## Configure Network Connection
+
+After the Wordpress stack is installed, we will need the AKS Cluster it
+provisions to be able to communicate with the MySQL database it provisions. In
+Azure, we can do so using a [Virtual Network Rule][vnet-rule]. However, the rule
```

Comment:
Nitpick: I have been using a pattern of prefixing provider link reference names with their provider, so I would use something like `azure-vnet-rule` here. The benefit is it makes it easy to tell which ones are provider links, even if you don't know much about their concepts.